com.ibm.wala.core/src/main/java/com/ibm/wala/ipa/slicer/SlicerUtil.java

```diff
@@ -38,10 +37,11 @@ private SlicerUtil() {}
   public static Statement findCallTo(CGNode n, String methodName) {
     IR ir = n.getIR();
     for (SSAInstruction s : Iterator2Iterable.make(ir.iterateAllInstructions())) {
-      if (s instanceof SSAInvokeInstruction) {
-        SSAInvokeInstruction call = (SSAInvokeInstruction) s;
+      if (s instanceof SSAAbstractInvokeInstruction) {
```

Comment:
This is a related change to avoid an unnecessary dependence on `SSAInvokeInstruction`, which does not work for the AST frontend